packages/carbon-graphs/dev/examples/controls/line.js

```diff
@@ -414,12 +414,10 @@ export const renderLineLegendTo = (id) => {
   );
   data.bindLegendTo = '#legendContainer';
   const lineTime = Carbon.api.graph(data);
-  lineTime.loadContent(
-    Carbon.api.line(getDemoData(`#${id}`, 'LINE_TIMESERIES').data[0]),
-  );
-  lineTime.loadContent(
-    Carbon.api.line(getDemoData(`#${id}`, 'LINE_TIMESERIES').data[2]),
-  );
+  const line1 = Carbon.api.line(getDemoData(`#${id}`, 'LINE_TIMESERIES').data[0]);
+  const line2 = Carbon.api.line(getDemoData(`#${id}`, 'LINE_TIMESERIES').data[2]);
+
+  lineTime.loadBatchContent([line1, line2])
```

Comment:
Do we need to update other graphs also to accept as an array?

Reply:
I have update examples in [this commit](https://github.com/cerner/terra-graphs/pull/29/commits/fe9ffede2d73d0394b5915fe5b860649af39a876). Note Pie graphs already used arrays for content, [pie example data](https://github.com/cerner/terra-graphs/blob/main/packages/carbon-graphs/dev/examples/data.js#L1602).